scripts/cmscp.py

```diff
@@ -1134,6 +1135,7 @@ def update_exit_info(exit_info, exit_code, exit_msg = None, force = False):
                              'CRAB_StageoutPolicy', \
                              'CRAB_Destination', \
                              'CRAB_Dest', \
+                             'CRAB_Final_Dest', \
```

Comment:
whatever the name, add 5 comment lines explaining what exactly each is,
do not force people to read code to find out from the way it is used